config/lib/mongoose.js

```diff
@@ -24,25 +25,28 @@ module.exports.connect = function (cb) {
 
```

Comment:
Removed. The ESLint rule [no-unused-vars is turned off](https://github.com/meanjs/mean/blob/master/.eslintrc.js#L36), probably because there are a lot of ESLint errors generated from this rule in our tests and other places :/ That can be addressed in a separate PR.